src/core/reaction_ensemble.hpp

```diff
@@ -309,7 +309,8 @@ class ConstantpHEnsemble : public ReactionAlgorithm {
 class WidomInsertion : public ReactionAlgorithm {
 public:
     double measure_excess_chemical_potential(int reaction_id);
-
+    int do_reaction(int reaction_steps)override {return 0;};
```

Comment:
This seems unrelated, was this on purpose?

Reply:
yep this is unrelated. do_reaction for the widom insertion method has no meaning. therefore i made it an empty function.